### PR TITLE
Describe the workings of the core engine.

### PR DESCRIPTION
**Pull request by pabzm**
An abstract to introduce into the DC core engine.

@hpk42, @r10s Could you verify that the description is correct?

### CODE REVIEW

**Review by r10s — APPROVED** [review]
good summary!

File: assets/main.css [hunk 1]
```diff
@@ -32,6 +32,10 @@ a {
   color: #666;
 }
 
+a * {
+  color: #666;
```
Comment (r10s) [inline assets/main.css:36]: 🤘

**Comment by ghost** [comment]
The link to the development page on staging is currently broken, I guess I have to touch the nginx config again. Here is the correct link for now: https://staging.bots.delta.chat/17/development.html - **edit: fixed.**

The code blocks don't look like code blocks yet. do we want to tackle this in this PR? I could also make an extra PR in the context of #2.

**Comment by ghost** [comment]
I'll open a different PR for the code blocks.

**Comment by ghost** [comment]
Correction: the code blocks look fine in production.

**Pull request merged**